### PR TITLE
users layout

### PR DESCRIPTION
**Pull request by serejabogdan**
created: Users;
added: table, pagination, dropdown, checkboxes

### CODE REVIEW

File: src/pages/private/users/index.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
+// outsource dependencies
+import React, { useEffect } from 'react';
+import Pagination from 'rc-pagination';
+import { useDispatch, useSelector } from 'react-redux';
+import {
```
Comment (ichabanivan) [inline src/pages/private/users/index.js]: Write import in one line

File: src/pages/private/users/index.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+// local dependencies
+import { TYPE, selector } from './reducer';
+import { setLocalStorage } from '../../../utils/local-storage';
+import { CURRENT_PAGE, USERS_NUMBER } from '../../../constants/local-storage';
+
+// styles
+import 'rc-pagination/assets/index.css';
+
+function fixDate (digit) {
```
Comment (ichabanivan) [inline src/pages/private/users/index.js]: use momentjs

File: src/pages/private/index.js [hunk 3]
```diff
@@ -30,6 +35,7 @@ function Private () {
         path={PRIVATE_HOMEPAGE}
         component={() => <Homepage user={user}/>}
```
Comment (sajera) [inline src/pages/private/index.js:36]: Should be used another ways to provide logged user to into Route nested components ?

File: src/pages/private/index.js [hunk 4]
```diff
@@ -30,6 +35,7 @@ function Private () {
...
       />
+      <Redirect to={PRIVATE_USERS}/>
```
Comment (sajera) [inline src/pages/private/index.js:38]: duplication of 26-29 ?

File: src/pages/private/reducer.js [hunk 5]
```diff
@@ -25,5 +31,10 @@ export default function privateReducer (state= initial, action) {
 }
 
 export function selector (state) {
-  return state.root.pages.private;
+  return state.root.pages.private.state;
```
Comment (sajera) [inline src/pages/private/reducer.js:34]: to much nesting within reducer

**Pull request merged**